SwissTransfer/Sources/SwissTransferApp.swift

```diff
@@ -29,12 +31,32 @@ struct SwissTransferApp: App {
     private let sentryService = SentryService()
     private let dependencyInjectionHook = TargetAssembly()
 
+    @LazyInjectService private var settingsManager: AppSettingsManager
+
+    @StateObject var appSettings: FlowObserver<AppSettings>
```

Comment:
State is private

Reply:
Yeah, passing most things in private is in my todo list